src/Spec/Swagger2.php

```diff
@@ -337,6 +338,18 @@ public function getDefinitions()
                     }
                 }
             }
+            if (isset($sch['error_types'])) {
+                $types = [];
+                foreach ($sch['error_types'] as $type) {
+
+                    $types[] = [
+                        'code' => $type['code'],
+                        'type' => $type['type'],
+                        'description' => $type['description']
+                    ];
+                }
+                $sch['error_types'] = $types;
+            }
```

Comment:
It would be good to move this into a separate PR that we can try to merge sooner and then all the other SDK templates would use that as the base.